eks-s3-seldon-mlflow/locals.tf

```diff
@@ -32,6 +32,15 @@ locals {
       name = "zenml-kubernetes"
       enable_container_registry = true
   }
+
+  rds = {
+    rds_name = "zenml-rds"
+    db_name = "zenmldb"
+    db_type = "mysql"
+    db_version = "8.0.28"
+    username = "admin"
+    password = "k8szenml"
```

Comment:
Isnt this slightly insecure?

Reply:
I have a few ideas.
- First, the user would not need to commit these values - just fork the repo, populate the locals file and run. But this is definitely not ideal. 
- The user can leave the password field empty. In that case, a random password will be created which can be accessed through a terraform command in the CLI - `terraform output metadata-db-password`
- Another way could be to have the user supply this value (and other secret value) during the script execution in the CLI. This will include removing the reference of these vars from the locals file. 

I like option 2 and 3 but we can't have them together. What do you think?